docs/release-process.md

```diff
@@ -0,0 +1,68 @@
+# Release Process
+
+When releasing code, we take the view that the `master` branch is always
+deployable. Whenever we merge a branch into `master`, this is automatically
+deployed to our development environment in the DfE Cloud Platform.
+
+To deploy to production, we take the following steps.
+
+## 1. Update the Changelog and create a pull request
+
+- Create a branch from `master` for the release called `release-x` where `x` is
+  the release number
```

Comment:
One last question: do we want to 0-pad the release number for alphabetizing? If so, 3 digits should be sufficient, I think.

Reply:
interesting, I hadn't considered that... we could I guess?

Reply:
It'll make it easier to tell what release we're on from a glance at the tags, and introduce fewer errors. I'm not particularly wedded either way, though.

Reply:
do it.

Reply:
Done.

Reply:
Great! Let's get this merged when it's green.